components/Text/Text.tsx

```diff
@@ -73,19 +73,29 @@ export const Text = styled('span', {
     },
     transform: {
       uppercase: {
-        textTransform: 'uppercase'
+        textTransform: 'uppercase',
```

Comment:
There are many lint changes, what about making sure we all have the same lint rules ?